sci-libs/hdf5/hdf5-1.12.1.ebuild

```diff
@@ -29,6 +29,10 @@ RESTRICT="!test? ( test )"
 
 RDEPEND="
 	mpi? ( virtual/mpi[romio] )
+	ros3? (
+		net-misc/curl
+		dev-libs/openssl
```

Comment:
Does it link to curl and openssl? If yes, we need to add `:=`

Reply:
@jsmolic I am pretty sure it doesn't (some other packages which depend on hdf5[ros3] fail some tests in the network sandbox and the error is clearly ascribed to the libs and not to hdf5). But how would I know for sure?

Reply:
@jsmolic as per the discussion just now both are linked but only openssl has multiple slots